.gitignore

```diff
@@ -6,11 +6,13 @@ provider.tf
 # .tfstate files
 *.tfstate
 *.tfstate.*
+.terraform.lock.hcl
```

Comment:
Please remove.

Reply:
why this should be removed?

Reply:
We discussed this earlier internally and we concluded that it was ok for examples but not required for the module itself.

Let's remove it for the purpose of this PR and discuss this again when everybody is back on deck.